src/sentry/utils/outcomes.py

```diff
@@ -28,6 +29,40 @@ class Outcome(IntEnum):
 outcomes_publisher = None
 
 
+def _get_outcome_cache_key(project_id, event_id):
+    return "otcm_{}_{}".format(project_id, event_id)
```

Comment:
Our default schema for such cache keys uses colons, for instance: `otcm:{}:{}`. Can you please change this so that it is consistent?